<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="rima-PC" name="cz.cuni.mff.d3s.deeco.test.unit.knowledge.LocalKnowledgeRepositoryTest" tests="3" failures="0" timestamp="5 Apr 2013 08:30:29 GMT" time="0.016" errors="0">
  <testcase name="get" time="0.000" classname="cz.cuni.mff.d3s.deeco.test.unit.knowledge.LocalKnowledgeRepositoryTest"/>
  <testcase name="take" time="0.000" classname="cz.cuni.mff.d3s.deeco.test.unit.knowledge.LocalKnowledgeRepositoryTest"/>
  <testcase name="getWithException" time="0.016" classname="cz.cuni.mff.d3s.deeco.test.unit.knowledge.LocalKnowledgeRepositoryTest"/>
</testsuite>
